app/activity/[title]/page.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+import {
+  Heading,
+  Box,
+  Container,
+  useColorModeValue,
+  Flex,
+  Spacer,
+  Text,
+  Textarea,
+  Input,
+  FormControl,
+  FormLabel
+} from '@chakra-ui/react'
+import { FiChevronLeft, FiClock, FiMapPin, FiLink2 } from 'react-icons/fi'
+import { Carousel } from '@/components/carousel'
+import { Link } from '@/components/link'
+import { Header, Footer } from '@/components/navigation'
+import { TrashIcon, EditIcon } from '@/icons'
+import { customColors } from '@/theme/color'
+
+export default function ActivityDetails({ params }: { params: { title: string } }) {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  return (
+    <>
+      <Header />
+      <Box as="main" minH="100vh" bg={bg} color={color}>
+        <Container
+          maxW={{ base: '710px' }}
+          pt={{ base: '20px', md: '30px' }}
+          pb={{ base: '40px', md: '80px' }}
+        >
+          <Flex pb={{ base: '30px', md: '40px' }}>
```

Comment:
Personally I do not recommend putting all HTML in pages because it's hard to maintain when you want to fix or something. I think it's better to divide to components if you can! Or you can simply just make 'ActivityDetails' component and import here. So that you can change it easily :)

Reply:
Ok, I divided some parts in components and imported them into the activity/id page. Let me know if that is what you meant!

Reply:
Thank you! I meant it's alway good to make components and gather them rather than lots of HTML in page.tsx.  There are some reasons. First is you can close the dependencies into smaller part and you can maintain more easily. Second one is meaning component is useful and easier for test!